CHANGELOG.md

```diff
@@ -7,10 +7,7 @@
 - 162c4e8: Spinner swap aria label for alt
 - 0b2a12f: add theming of navbar (desktop)
 
```

Comment:
did you edit this manually?

Reply:
there was a conflict.. so fixed on ui. what would you normally do?

Reply:
Yeah don't touch this file basically - I'd revert it back to the CHANGELOG pre this commit

Reply:
Good to know..